rs/ic-management-types/src/lib.rs

```diff
@@ -563,6 +564,16 @@ pub enum Network {
     Url(url::Url),
 }
 
+impl Debug for Network {
```

Comment:
Network already implements Display so you should be able to use it instead of rewriting.

Reply:
Since `Cli` derives off of `Debug` I need it explicitly

Reply:
it's ok, just call .display() instead of providing manual values.